platforms/documentation/docs/src/docs/userguide/reference/glossary.adoc

```diff
@@ -36,26 +55,36 @@ They comprise <<#sub:terminology_artifact,artifacts>> as well as the appropriate
 For example, the `java` component in its default setup consists of a JAR — produced by the `jar` task — and the dependency information of the Java _api_ and _runtime_ variants.
 It may also define additional variants, for example, _sources_ and _Javadoc_, with the corresponding artifacts.
 
-Component metadata rule::
+Component Metadata Rule::
 A component metadata rule is a rule that modifies a component's metadata after it is fetched from a repository, e.g., to add missing information or correct incorrect information.
 In contrast to <<#sub:terminology_resolution_rule, resolution rules>>, component metadata rules are applied *before* resolution starts.
 Component metadata rules are defined as part of the build logic and can be shared through plugins.
 For more information, see the section on <<component_metadata_rules.adoc#component-metadata-rules,fixing metadata with component metadata rules>>.
 
 [[sub:terminology_configuration]]
-Configuration ::
+Configuration::
 A configuration is a named set of <<#sub:terminology_dependency, dependencies>> grouped together for a specific goal.
 Configurations provide access to the underlying, resolved <<#sub:terminology_module, modules>> and their artifacts.
 For more information, see the sections on <<dependency_configurations.adoc#sub:what-are-dependency-configurations,dependency configurations>> as well as <<declaring_configurations.adoc#sec:resolvable-consumable-configs,resolvable and consumable configurations>>.
 +
 NOTE: The word "configuration" is an overloaded term that has a different meaning outside of dependency management.
 
-[[sub:cross_configuration]]
-Cross-Configuration ::
-See <<sub:cross_project_configuration,Cross-Project Configuration>>.
+[[sub:terminology_configuration_phase]]
+Configuration Phase::
+A Gradle build is composed of two primary phases, the configuration phase (not to be confused with <<sub:terminology_configuration>> instances) and the <<sub:terminology_execution_phase,execution phase>>.
++
+The configuration phase happens first and is single-threaded.
+
+[[sub:terminology_convention_plugin]]
+Convention Plugin::
+A <<sub:terminology_plugin,plugin>> built on top of an <<sub:terminology_ecosystem_plugin,ecosystem plugin>>, which applies common _conventions_ to the build script that uses the plugin.
 
-[[sub:cross_project_configuration]]
```

Comment:
💅 consider preserving old anchors in case somebody refers them externally

Reply:
I typically do but in this case the anchor was poorly worded and should have *terminology* in it.
Luckily this is a relatively new anchor so it will be low impact.